templates/azure-pipelines.yml

```diff
@@ -232,54 +235,54 @@ stages:
 
   - job: 'macOS'
     pool:
-      vmImage: 'macOS-10.13'
+      vmImage: 'macOS-10.14'
     strategy:
       matrix:
-        Xcode 10.1:
+        Xcode 11.3.1:
           B2_TOOLSET: clang
           B2_CXXSTD: 14,17,2a
-          XCODE_APP: /Applications/Xcode_10.1.app
-        Xcode 10.0:
+          XCODE_APP: /Applications/Xcode_11.3.1.app
+        Xcode 11.2.1:
           B2_TOOLSET: clang
           B2_CXXSTD: 14,17,2a
-          XCODE_APP: /Applications/Xcode_10.app
-        Xcode 9.4.1:
-          B2_TOOLSET: clang
-          B2_CXXSTD: 11,14,17
-          XCODE_APP: /Applications/Xcode_9.4.1.app
-        Xcode 9.4:
+          XCODE_APP: /Applications/Xcode_11.2.1.app
+        Xcode 11.2:
           B2_TOOLSET: clang
-          B2_CXXSTD: 11,14,17
-          XCODE_APP: /Applications/Xcode_9.4.app
-        Xcode 9.3.1:
+          B2_CXXSTD: 14,17,2a
+          XCODE_APP: /Applications/Xcode_11.2.app
+        Xcode 11.1:
           B2_TOOLSET: clang
-          B2_CXXSTD: 11,14,17
-          XCODE_APP: /Applications/Xcode_9.3.1.app
-        Xcode 9.3:
+          B2_CXXSTD: 14,17,2a
+          XCODE_APP: /Applications/Xcode_11.1.app
+        Xcode 10.3:
           B2_TOOLSET: clang
-          B2_CXXSTD: 11,14
-          XCODE_APP: /Applications/Xcode_9.3.app
-        Xcode 9.2:
+          B2_CXXSTD: 14,17,2a
+          XCODE_APP: /Applications/Xcode_10.3.app
+        Xcode 10.2.1:
           B2_TOOLSET: clang
-          B2_CXXSTD: 11,14
-          XCODE_APP: /Applications/Xcode_9.2.app
-        Xcode 9.1:
+          B2_CXXSTD: 14,17,2a
```

Comment:
Why are we removing C++11 ?

Reply:
Currently there is one remaining C++11 test, for 9.4.1.  The diff presentation is misleading, what's actually happening is all of these are being removed since they don't exist in the macOS-10.14 image:

https://github.com/boostorg/boost-ci/blob/19c177d77c98fcf6e203d5478871632e439170ad/templates/azure-pipelines.yml#L250-L281

The 9.4.1 build broke though, stating [no C++11 compiler was found](https://dev.azure.com/svenevs/boost/_build/results?buildId=391&view=logs&j=1af78d47-08f5-5e8f-f082-3b40a6afd3ae&t=6d964bff-0505-5c9a-0b06-eaeef7dffb77&l=359)...

We could add C++11 back in to one of the newer Xcode though, e.g., 10 series?

Reply:
I don't understand. Are you saying that the macOS-10.14 image does not have C++11 support?

Reply:
Sorry for not being clear, there are two separate things:

1. Large number of compilers no longer available, all are being removed.  The only one was previously being used on 10.13 image with C++11 is the 9.4.1 version (earliest available in new image).
2. I could not get the 9.4.1 build to work on the different repos I was trying to test this on.

So the question really is, assuming 9.4.1 somehow resolves or I was not testing correct repo, is only 1 C++11 compile on macOS ok?  Or should we add 11 to B2_CXXSTD for some other XCode versions in addition?

Reply:
Resolved in 8c4b257, see https://github.com/boostorg/boost-ci/pull/42#issuecomment-605309149

Reply:
Regarding the unknown problems with 9.4.1m I've just come across this https://github.com/Microsoft/azure-pipelines-image-generation/issues/886#issuecomment-487101917, copying

```
/bin/bash -c sudo xcode-select --switch /Applications/Xcode_9.4.1.app/Contents/Developer

xcodebuild -showsdks
macOS SDKs:
macOS 10.13 -sdk macosx10.13
```